Source/Filters/YPPhotoFiltersVC.swift

```diff
@@ -118,48 +115,68 @@ open class YPPhotoFiltersVC: UIViewController, IsMediaFilterVC, UIGestureRecogni
     
     @objc
     func save() {
```

Comment:
Only processing the full image at the end on save is also a great optimization 👏

Reply:
Thanks)